phpBB/styles/prosilver/template/posting_buttons.html

```diff
@@ -97,7 +97,11 @@
 
 	<!-- BEGIN custom_tags -->
 	<button type="button" class="button button-secondary bbcode-{custom_tags.BBCODE_TAG_CLEAN}" name="addbbcode{custom_tags.BBCODE_ID}" value="{custom_tags.BBCODE_TAG}" onclick="bbstyle({custom_tags.BBCODE_ID})" title="{{ custom_tags.BBCODE_HELPLINE }}">
-		{custom_tags.BBCODE_TAG}
+		<!-- IF custom_tags.BBCODE_TAG != '' -->
+			<i class="icon fa-{custom_tags.BBCODE_ICON_NAME} fa-fw" aria-hidden="true"></i>
+		<!-- ELSE -->
+			{custom_tags.BBCODE_TAG}
+		<!-- ENDIF -->
```

Comment:
I think you should use twig syntax.

Reply:
I disagree with this. I specifically matched the tags already in this file.